src/workers/event.rs

```diff
@@ -944,9 +950,12 @@ where
                         {
                             warn!(error = ?e, "Failed to set status to undeployed");
                         }
-                        Ok(None)
+                        return message.ack().await.map_err(WorkError::from);
+                    }
+                    Some(Err(e)) => {
+                        message.nack().await;
+                        return Err(WorkError::Other(e.into()));
                     }
-                    Some(Err(e)) => Err(e),
```

Comment:
This is an optimization, no scalers need to handle the ManifestUnpublished message so we can just ack / nack here instead of running _every_ scaler.